crowbar_framework/app/controllers/repositories_controller.rb

```diff
@@ -64,12 +95,10 @@ def sync
     redirect_to repositories_path
   end
 
-  #
-  # Activate a single Repository
-  #
-  # Provides the restful api call for
-  # Activate a Repository   /utils/repositories/activate   POST  Creates Repository DataBagItem
-  # required parameters: platform, arch, repo
+  api :POST, "/utils/repositories/activate", "Activate a single repository. Creates a DataBagItem"
+  param :platform, String, desc: "Platform of the repository", required: true
+  param :arch, String, desc: "Architecture of the repository", required: true
+  param :repo, String, desc: "Name of the repository", required: true
```

Comment:
HTTP Codes returned needs also to be documented. Later on, you can also use http codes for specific backend errors, when described